def exit_with_error_msg(msg):
    """Function prints error message and exists program with value 1.
        Parameters: error message string
    """
    print("EXIT: %s" % msg)
    exit(1)
